packages/toolkit/src/shared/sam/cli/samCliDetection.ts

```diff
@@ -26,14 +26,10 @@ const settingsUpdated = localize('AWS.samcli.detect.settings.updated', 'Settings
  */
 export async function detectSamCli(args: { passive: boolean; showMessage: boolean | undefined }): Promise<void> {
     const config = SamCliSettings.instance
-
-    if (!args.passive) {
-        // Force getOrDetectSamCli() to search for SAM CLI.
-        await config.delete('location')
-    }
```

Comment:
remove call to `config.delete('location')` since `getOrDetectSamCli()` gained the `forceSearch` param in:
https://github.com/aws/aws-toolkit-vscode/commit/695ad0d263bc65217f77e83c40f42e87c5c70435